src/code42cli/cmds/auditlogs.py

```diff
@@ -0,0 +1,255 @@
+import json
+from datetime import datetime
+from datetime import timezone
+from datetime import timedelta
+from _collections import OrderedDict
+
+import click
+
+from code42cli.click_ext.groups import OrderedGroup
+from code42cli.cmds.search.cursor_store import AuditLogCursorStore
+from code42cli.cmds.search.options import BeginOption
+from code42cli.date_helper import parse_max_timestamp
+from code42cli.date_helper import parse_min_timestamp
+from code42cli.logger import get_logger_for_server
+from code42cli.options import begin_option
+from code42cli.options import end_option
+from code42cli.options import format_option
+from code42cli.options import sdk_options
+from code42cli.options import send_to_format_options
+from code42cli.options import server_options
+from code42cli.output_formats import OutputFormatter
+from code42cli.util import warn_interrupt
+
+EVENT_KEY = "events"
+AUDIT_LOGS_KEYWORD = "audit-logs"
+AUDIT_LOG_TIMESTAMP_FORMAT = "%Y-%m-%dT%H:%M:%S.%f"
+
+AUDIT_LOGS_DEFAULT_HEADER = OrderedDict()
+AUDIT_LOGS_DEFAULT_HEADER["timestamp"] = "Timestamp"
+AUDIT_LOGS_DEFAULT_HEADER["type$"] = "Type"
+AUDIT_LOGS_DEFAULT_HEADER["actorName"] = "ActorName"
+AUDIT_LOGS_DEFAULT_HEADER["actorIpAddress"] = "ActorIpAddress"
+AUDIT_LOGS_DEFAULT_HEADER["userName"] = "AffectedUser"
+AUDIT_LOGS_DEFAULT_HEADER["userId"] = "AffectedUserUID"
+# AUDIT_LOGS_DEFAULT_HEADER["success"] = "Success"
+# AUDIT_LOGS_DEFAULT_HEADER["resultCount"] = "ResultCount"
+
+filter_option_usernames = click.option(
```

Comment:
Is this a c42-username?  Might be worth sharing some of these filter options. File Events has a --c42-username and a I think it could potentially make use of a user-id filter (though it doesn't currently).